clang/lib/DPCT/Error.cpp

```diff
@@ -38,12 +38,17 @@ void ShowStatus(int Status, std::string Message) {
     StatusString = "Error: Path for CUDA header files specified by "
                    "--cuda-include-path is invalid.";
     break;
+  case MigrationErrorSupportedCudaVersionNotAvailable:
+    StatusString = "Error: The auto detect CUDA header files version is not supported. "
+                  "See Release Notes for supported versions. "
```

Comment:
let's be more specific/clear on the version range supported by specifiy.  "this version of syclomatic support ...."